cmd/git-chglog/config.go

```diff
@@ -308,7 +308,7 @@ func (config *Config) Convert(ctx *CLIContext) *chglog.Config {
 		Options: &chglog.Options{
 			NextTag:                     ctx.NextTag,
 			TagFilterPattern:            ctx.TagFilterPattern,
-			Sort:                        opts.Sort,
+			Sort:                        orValue(ctx.Sort, opts.Sort),
```

Comment:
Nice. This was what I was looking for. 😀

Reply:
Updated 😄